cluster/manifests/03-ebs-csi/02-csi-driver.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: csi-node-driver
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      application: container-storage-interface
+      component: driver
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        application: container-storage-interface
+        component: driver
+    spec:
+      containers:
+        - args:
+            - node
+            - --endpoint=$(CSI_ENDPOINT)
```

Comment:
Didn't know you could do that!

Reply:
Can be used for when the application cannot accept a secret through a mounted environment variable.